tex/gregoriotex.lua

```diff
@@ -214,7 +214,8 @@ local function init(arg, enable_height_computation)
   if enable_height_computation then
     new_line_heights = {}
 
-    local mcb_version = luatexbase.get_module_version('luatexbase-mcb')
+    local mcb_version = luatexbase.get_module_version and
+        luatexbase.get_module_version('luatexbase-mcb') or 9999
```

Comment:
Probably because most other languages actually have a ternary operator or a functional-if and don't need such an idiom.  I've seen shell (which also has neither) use this idiom as well.